src/openforms/prefill/contrib/communication_preferences/config.py

```diff
@@ -0,0 +1,29 @@
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import serializers
+
+from openforms.api.fields import SlugRelatedAsChoicesField
+from openforms.contrib.customer_interactions.models import (
+    CustomerInteractionsAPIGroupConfig,
+)
+from openforms.formio.api.fields import FormioVariableKeyField
+from openforms.utils.mixins import JsonSchemaSerializerMixin
+
+
+class ProfileCommunicationPreferencesSerializer(
```

Comment:
I think its best to stick to one naming for the prefill plugin parts
```suggestion
class CommunicationPreferencesSerializer(
```